lib/rds_sql_server/rds.rb

```diff
@@ -34,11 +34,17 @@ def self.timeout
     @timeout || 50_000_000
   end
 
+  def self.rds_available?
+    new.client.present?
+  end
+
   def initialize
     self.client = Aws::RDS::Client.new
+  rescue RuntimeError
+    self.client = nil
   end
 
-  define_method(:sqlservers) { _list.select { |server| server.engine.match(/sqlserver/) } }
+  define_method(:sqlservers) { _list&.select { |server| server.engine.match(/sqlserver/) } }
```

Comment:
I wonder why this uses define_method in this class. The methods don't appear to be dynamic
```suggestion
  def sqlservers
    _list&.select { |server| server.engine.match(/sqlserver/) }
  end
```

Reply:
@yakloinsteak, this is something you wrote (years ago).  Any chance you remember why you went with `define_method` instead of `def`?  I feel like maybe it had to do with the dynamic loading and the lack of always having an RDS connection to rely on, so maybe it got unhappy at compilation time if they were static method definitions?

Reply:
`_list` might have been a variable from above that method in some previous iteration, but I've also been known to use `define_method` for one-line methods to save space. I don't think we need it, and what @ttoomey sounds fine to me.

Reply:
Thanks!